jokers/satellite.lua

```diff
@@ -0,0 +1,29 @@
+﻿return {
+    key = "satellite",
+    base_rate = folly_utils.stack(),
+    add_to_deck = function(self, card, from_debuff)
+        self.base_rate:push(G.GAME.planet_rate)
+        G.GAME.planet_rate = 0;
```

Comment:
this is sadly negated by buying a planet voucher...

this also has the side effect of lowering your planet rate if you sell this joker after buying the planet merchant voucher